packages/create-pantheon-decoupled-kit/src/templates/next-wp/pages/posts/[...slug].jsx.hbs

```diff
@@ -3,22 +3,22 @@ import { ContentWithImage } from '@pantheon-systems/nextjs-kit';
 import { setOutgoingHeaders } from '../../lib/setOutgoingHeaders';
 
 import Layout from '../../components/layout';
-import PageHeader from '../../components/page-header';
-
-import { PostGridItem } from '../../components/grid';
-import { withGrid } from '@pantheon-systems/nextjs-kit';
-
 import { getFooterMenu } from '../../lib/Menus';
+import { IMAGE_URL } from '../../lib/constants';
 import { getPostByUri, getPostPreview } from '../../lib/Posts';
+{{#if wpAcfAddon}}
+import { PostGrid } from '../../components/grid';
+{{/if}}
+{{#unless tailwindcss}}
+import styles from './[...slug].module.css';
```

Comment:
I think wires are crossed somehow here. When I create a starter kit, viewing posts fail because this css module doesn't exist. The actual hbs template seems to only apply if the acf addon is being used. So maybe the conditional is wrong here?

Reply:
Looks like the conditional was wrong, but even when it is correct, this issue highlights the fact that this pattern with an empty `templates` array in the generator won't really work for the add-ons. I am breaking out these files into partials.